Makefile

```diff
@@ -10,14 +10,14 @@ OS := $(shell sh -c 'uname -s 2>/dev/null')
 BASE_ASSETS = $(shell find assets/base -type d) $(shell find assets/base -type f -name '*')
 OVERRIDE_ASSETS = $(shell find assets/override -type d 2> /dev/null) $(shell find assets/override -type f -name '*' 2> /dev/null)
 
-.yarninstall: package.json
-	@if ! [ $(shell which yarn 2> /dev/null) ]; then \
-		echo "yarn is not installed https://yarnpkg.com"; \
+.npminstall: package.json
+	@if ! [ $(shell which npm 2> /dev/null) ]; then \
+		echo "npm is not installed https://npmjs.com"; \
 		exit 1; \
 	fi
 
 	@echo Getting Javascript dependencies
-	@yarn install --pure-lockfile
+	@npm install
```

Comment:
There is no flag needed so it uses the exact commits like with yarn?

Reply:
Correct. `npm install` will only modify the lockfile if the package.json was changed, so it should install the exact versions from the lockfile